neo/test/rawiotest/test_phyrawio.py

```diff
@@ -0,0 +1,48 @@
+"""
+Tests of neo.rawio.examplerawio
+
+Note for dev:
+if you write a new RawIO class your need to put some file
+to be tested at g-node portal, Ask neuralensemble list for that.
+The file need to be small.
+
+Then you have to copy/paste/renamed the TestExampleRawIO
+class and a full test will be done to test if the new coded IO
+is compliant with the RawIO API.
+
+If you have problems, do not hesitate to ask help github (prefered)
+of neuralensemble list.
+
+Note that same mechanism is used a neo.io API so files are tested
+several time with neo.rawio (numpy buffer) and neo.io (neo object tree).
+See neo.test.iotest.*
+
+
+Author: Samuel Garcia
+
```

Comment:
You can remove this

Reply:
Yes, I will fix/edit docstrings, PEP8 and add unit tests for tsv/csv parser. However, I don't think I'll be able to do it today.

Reply:
Removed